### PR TITLE
Provide better NOTES text for MLS

### PR DESCRIPTION
**Pull request by mkysel**

### CODE REVIEW

**Review by fbac — APPROVED** [review]

File: helm/mls-validation-service/templates/NOTES.txt [hunk 1]
```diff
@@ -1 +1,18 @@
-1. Get the application URL by running these commands:
+###########################################################
+###                 Application Access Info             ###
+###########################################################
+
+Your application has been deployed successfully!
+
+You can access it using the following command:
```
Comment (jhaaaa) [inline helm/mls-validation-service/templates/NOTES.txt:7]:
```suggestion
You can access it using the following commands:
```

File: helm/mls-validation-service/templates/NOTES.txt [hunk 2]
```diff
@@ -1 +1,18 @@
...
+
+To get the ClusterIP service:
```
Comment (jhaaaa) [inline helm/mls-validation-service/templates/NOTES.txt:9]:
```suggestion
To get the ClusterIP service, run:
```

File: helm/mls-validation-service/templates/NOTES.txt [hunk 3]
```diff
@@ -1 +1,18 @@
...
+  export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ include "mls-validation-service.fullname" . }} -o jsonpath='{.spec.clusterIP}')
+  echo "Application is available at http://$SERVICE_IP:{{ .Values.service.port }}"
+
+If you are using port-forwarding:
```
Comment (jhaaaa) [inline helm/mls-validation-service/templates/NOTES.txt:13]:
```suggestion
If you are using port-forwarding, run:
```

**Review by jhaaaa — APPROVED** [review]
Thank you for this @mkysel! Suggesting just a few 🤓 tweaks....

**Pull request merged**